src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/mturk/MturkPlatform.java

```diff
@@ -88,41 +89,32 @@ public Boolean isCalibrationAllowed() {
         });
     }
 
-    private static List<String> loadFiles(List<String> files) {
-        return files.stream().map(s -> {
-            try {
-                return CharStreams
-                        .toString(new InputStreamReader(
-                                Main.class.getResourceAsStream(s)
-                                , Charsets.UTF_8));
-            } catch (IOException e) {
-                e.printStackTrace();
-                return "";
-            }
-        })
-        .filter(s1 -> !s1.isEmpty())
-        .collect(Collectors.toList());
+    private static String loadFiles(String file) {
+        try {
+            return CharStreams
+                    .toString(new InputStreamReader(
+                            Main.class.getResourceAsStream(file)
+                            , Charsets.UTF_8));
```

Comment:
may I ask why ? in this case it makes the code longer, since a have to join the list of lines to one long line ...